### DISCUSSION TITLE
Custome Window Sizes

### DISCUSSION THREAD

**Discussion by gerootech** [rxhanson/Rectangle#517] [Q&A]
Is it possible to create custom window size and assign it to a shortcut ?

**Comment by rxhanson** [comment]
You can add an extra centering command using terminal commands:
https://github.com/rxhanson/Rectangle#add-an-extra-centering-command-with-custom-size

Or you could adjust the almost maximize size using terminal commands:
https://github.com/rxhanson/Rectangle#almost-maximize

Beyond that, there's my paid superset of Rectangle, [Hookshot](https://hookshot.app), that has a custom shortcut builder where you can specify window dimensions and where the windows are docked. It also allows you to specify different window dimensions/locations for repeated presses of the custom shortcut. Additionally, you can set custom shortcuts to a direction in the main Hookshot functionality in the app.

The last way to do this would just be to fork Rectangle and write your own.
